sample/development/components/integration/test/integrationTests.brs

```diff
@@ -109,6 +109,211 @@ function TS_SDK_integration() as object
             return validator
         end function,
 
+        TC_SDK_resetIdentities_clearsLocationHintAndStateStore: function() as dynamic
+            aepSdk = ADB_retrieveSDKInstance()
+            ADB_CONSTANTS = AdobeAEPSDKConstants()
+
+            configuration = {}
+            configuration[ADB_CONSTANTS.CONFIGURATION.EDGE_CONFIG_ID] = m.configId
+
+            aepSdk.setLogLevel(ADB_CONSTANTS.LOG_LEVEL.VERBOSE)
+
+            ' API call 1
+            aepSdk.updateConfiguration(configuration)
+            eventIdForUpdateConfiguration = aepSdk._private.lastEventId
+
+            ' API call 2
+            aepSdk.sendEvent({
+                "xdm": {
+                    "key": "value"
+                }
+            })
+            eventIdForSendEvent1 = aepSdk._private.lastEventId
+
+            ' API call 3
```

Comment:
Is there a reason you’re sending 2 events before resetIdentities? It’s making the test case quite large.

Reply:
@praveek 
It is just to show the second request goes with Location Hint and has state:store in the request payload.